src/db/spanner/models.rs

```diff
@@ -1204,7 +1204,7 @@ impl SpannerDb {
             Sorting::Index => format!("{} ORDER BY sortindex DESC", query),
             Sorting::Newest => format!("{} ORDER BY modified DESC", query),
             Sorting::Oldest => format!("{} ORDER BY modified ASC", query),
```

Comment:
```suggestion
            Sorting::Oldest | Sorting::None => format!("{} ORDER BY modified ASC", query),
```

but I'm wondering if we should just kill Sorting::None entirely (it would come in as Newest)

Reply:
I'd rather have Sorting::None state still be there. This code is confusing enough as it is.